Detectors/CTP/simulation/src/Digitizer.cxx

```diff
@@ -69,12 +69,23 @@ std::vector<CTPDigit> Digitizer::process(const gsl::span<o2::ctp::CTPInputDigit>
           break;
         }
         case o2::detectors::DetID::EMC: {
-          for (auto const& ctpinp : det2ctpinp[o2::detectors::DetID::EMC]) {
-            uint64_t mask = (inp->inputsMask).to_ullong() & detInputName2Mask[ctpinp.name];
-            if (mask) {
-              inpmaskcoll |= std::bitset<CTP_NINPUTS>(ctpinp.inputMask);
+          // uint64_t inpmaskdebug = 1;
+          uint64_t inpmaskdebug = (inp->inputsMask).to_ullong();
+          if (inpmaskdebug & detInputName2Mask["EMBA"]) {
```

Comment:
I am not sure this is needed but ok - you did debugg

Reply:
It is, because this is a fake intput. For min. bias we never enter this loop
```c++
for (auto const& ctpinp : det2ctpinp[o2::detectors::DetID::EMC]) {
  ...   
}
```
because the input is not part of the CTP configuration (det2ctpinp).

Reply:
Definitely it was not, I have checked that the loop was never entered. And this makes perfect sense, since `det2ctpinp` comes from the trigger configuration in the CCDB, so unless you modified the entries for all runs there it is not added.

Reply:
Yes, sorry, I thought that
https://github.com/AliceO2Group/AliceO2/blob/6dc7cb5f735131339ef66439af43524178e5c293/DataFormats/Detectors/CTP/src/Configuration.cxx#L1052
is used.